ui/src/features/Editor/components/Canvas/useDnd.ts

```diff
@@ -56,6 +56,7 @@ export default ({
         id: randomID(),
         position,
         data: {
+          name: "sample",
```

Comment:
This code isn't boilerplate or temp so "sample" isn't appropriate.
Is there a reason you added this specifically?

Reply:
I made this change in the types 
![image](https://github.com/user-attachments/assets/60fc4ce2-e180-4cf2-8d92-8763486d3d78)

Since node can't exist without name otherwise the endpoint `/actions/{name}` won't work 

Due to this change I had added `sample`  but I changed it to `""` to make more sense, the code after this line always add a `name` field so it's always overriden. (check the image below)

![image](https://github.com/user-attachments/assets/af7365a8-f99f-4c50-843a-8a4950783c07)